themes/bootstrap3/templates/search/searchbox.phtml

```diff
@@ -1,12 +1,25 @@
 <?php
-    // Set default value if necessary:
-    if (!isset($this->searchClassId)) {
-        $config = $this->config()->get('config');
-        $this->searchClassId = $config->Site->defaultSearchBackend ?? 'Solr';
+    // Initialize from current search (if available) or defaults:
+    if ($results = ($this->results ?? $this->searchMemory()->getCurrentSearch())) {
+      $params = $results->getParams();
+      $this->searchClassId = $params->getSearchClassId();
+      $hiddenFilters = $params->getHiddenFilters();
+      $lastSort = $params->getSort();
+      $lastLimit = $params->getLimit();
+      $options = $results->getOptions();
+    } else {
+      if (!isset($this->searchClassId)) {
+          $config = $this->config()->get('config');
+          $this->searchClassId = $config->Site->defaultSearchBackend ?? 'Solr';
+      }
+      $options = $this->searchOptions($this->searchClassId);
+      $hiddenFilters = $this->searchTabs()->getHiddenFilters($this->searchClassId, true, $this->ignoreHiddenFiltersInRequest ?? false);
+      $lastSort = null;
```

Comment:
Should these be defaulting to the last remembered sort/limit from the searchMemory instead of null? It looks to me like my last sort is no longer being remembered if I go back to the search home page, which is a change in behavior from the dev branch which I assume is not intentional.

Reply:
Yes, fixed.